tools/build_TizenRT/Makefile

```diff
@@ -63,11 +63,12 @@ CSRCS		= src/libnnstreamer-edge/nnstreamer-edge-data.c \
 		src/libnnstreamer-edge/nnstreamer-edge-util.c
 
 CFLAGS += -I ./include -DDEBUG=0
+CFLAGS += -I ../mosquitto
```

Comment:
can you check config for mqtt `CONFIG_NETUTILS_MQTT`?

1. Need to add dependency to mqtt
https://github.com/nnstreamer/nnstreamer-edge/blob/main/tools/build_TizenRT/Kconfig
```
	default n
	depends on NET && MQTT
```

See below code as ref,
https://github.com/Samsung/TizenRT/blob/master/os/net/Kconfig
```
source "../framework/src/network/mqtt/Kconfig.protocol"
```

https://github.com/Samsung/TizenRT/blob/master/external/mosquitto/Make.defs
```
ifeq ($(CONFIG_NETUTILS_MQTT),y)
CONFIGURED_EXT += mosquitto
endif
```

Reply:
Thanks. Updated.